### PR TITLE
📓 Updated HUN

### PR DESCRIPTION
**Pull request by KristofKekesi**

### CODE REVIEW

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 1]
```diff
@@ -2202,31 +2196,31 @@
...
         </trans-unit>
         <trans-unit id="BaseLayoutContextFlyoutUnpinDirectoryFromSidebar.Text" translate="yes" xml:space="preserve">
           <source>Unpin directory from sidebar</source>
-          <target state="new">Unpin directory from sidebar</target>
+          <target state="translated">Könyvtár levétele az oldalsávról </target>
         </trans-unit>
         <trans-unit id="StatusCenterFlyout.AutomationProperties.Name" translate="yes" xml:space="preserve">
           <source>Ongoing tasks flyout</source>
-          <target state="new">Ongoing tasks flyout</target>
+          <target state="new">Folyamatban lévő feladatok flyout</target>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.hu-HU.xlf:2207]: Can you update the state?

**Review by yaichenbaum — CHANGES_REQUESTED** [review]

File: Files/MultilingualResources/Files.hu-HU.xlf (hunk 1 above)
Comment (KristofKekesi) [reply Files/MultilingualResources/Files.hu-HU.xlf:2207]: I would like to keep the new status becouse its not the best and I want to change it later.

**Review by yaichenbaum — DISMISSED** [review]

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 2]
```diff
@@ -2078,11 +2072,11 @@
         </trans-unit>
         <trans-unit id="BundlesWidgetAddBundleErrorAlreadyExists" translate="yes" xml:space="preserve">
           <source>Bundle with the same name already exists!</source>
-          <target state="new">Bundle with the same name already exists!</target>
+          <target state="translated"Csomag ezzel a névvel már létezik!</target>
```
Comment (XPoppyX) [inline Files/MultilingualResources/Files.hu-HU.xlf:2075]: Missing ">" here

Captura de pantalla 2021-01-29 182502

Comment (KristofKekesi) [reply Files/MultilingualResources/Files.hu-HU.xlf]: Oops. Sorry I fixed it.

**Review by yaichenbaum — APPROVED** [review]

**Pull request merged**